src/modules/AmidoBuild/exported/Build-DockerImage.ps1

```diff
@@ -132,16 +138,27 @@ function Build-DockerImage() {
         }
         # Run command to login to the docker registry to do the push
         # The Invoke-External function will need to be updated to obfruscate sensitive information
+        Write-Information -MessageData ("Logging in")
         Invoke-External -Command $cmd
 
         if ($LASTEXITCODE -ne 0) {
             exit $LASTEXITCODE
         }
 
-        # Finally push the image
+        # Push the image with the desired tag
         $cmd = "docker push {0}/{1}:{2}" -f $registry, $name, $tag
+        Write-Information -MessageData ("Running: {0}" -f $cmd)
         Invoke-External -Command $cmd
 
+        # Push the image with the latest tag if latest flag is declared
+        if ($latest.IsPresent) {
+            $cmd = "docker push {0}/{1}:latest" -f $registry, $name
+            Write-Information -MessageData ("Running: {0}" -f $cmd)
```

Comment:
This whole cmd --> Write-Information --> Invoke-External is repeated at least one (and potentially in other files?)
I wonder if we need a method that takes a cmd string and does both lines in one?

Reply:
Good shout - amending Invoke-External to include this block instead?